client/blocks/inline-help/contextual-help.js

```diff
@@ -3,749 +3,866 @@
  * External dependencies
  */
 import { compact, first, get } from 'lodash';
+import { translate } from 'i18n-calypso';
 
 /**
  * Internal Dependencies
  */
 import { RESULT_TOUR, RESULT_VIDEO } from './constants';
+import { localizeUrl } from 'lib/i18n-utils';
 
 /**
  * Module variables
  */
 const fallbackLinks = [
 	{
-		link: 'https://en.support.wordpress.com/do-i-need-a-website-a-blog-or-a-website-with-a-blog/',
+		link: localizeUrl(
+			localizeUrl(
+				'https://en.support.wordpress.com/do-i-need-a-website-a-blog-or-a-website-with-a-blog/'
+			)
+		),
 		post_id: 143180,
-		title: 'Do I Need a Website, a Blog, or a Website with a Blog?',
-		description:
-			'If you’re building a brand new site, you might be wondering if you need a website, a blog, or a website with a blog. At WordPress.com, you can create all of these options easily, right in your dashboard.',
+		title: translate( 'Do I Need a Website, a Blog, or a Website with a Blog?' ),
+		description: translate(
+			'If you’re building a brand new site, you might be wondering if you need a website, a blog, or a website with a blog. At WordPress.com, you can create all of these options easily, right in your dashboard.'
+		),
 	},
 	{
-		link: 'https://en.support.wordpress.com/business-plan/',
+		link: localizeUrl( localizeUrl( 'https://en.support.wordpress.com/business-plan/' ) ),
```

Comment:
Is the double `localizeUrl()` intentional? Actually, we don't need it here as the localizeUrl already happens in https://github.com/Automattic/wp-calypso/pull/28166/files#diff-9d01b2007d686a6ec3241f54108060d8R53

Reply:
Eeek! Bad regex!